src/main/scala/au/com/agiledigital/toolform/command/generate/kubernetes/minikube/GenerateMinikubeCommand.scala

```diff
@@ -89,4 +91,10 @@ object GenerateMinikubeCommand extends YamlWriter {
 
   private def shouldWriteService(toolFormService: ToolFormService): Boolean =
     toolFormService.exposedPorts.nonEmpty || toolFormService.externalPorts.nonEmpty
+
+  private def isdiskResourceType(resource: Resource): Boolean =
```

Comment:
D

Reply:
I noticed and changed that just before i read your comment! Thanks, its been changed to proper camel case now